packages/channel-client/tests/fakes/fake-channel-provider.ts

```diff
@@ -1,17 +1,18 @@
-import {ChannelProviderInterface} from '@statechannels/channel-provider/src';
+import {ChannelProviderInterface, MethodType} from '@statechannels/channel-provider/src';
```

Comment:
Might be worth moving this file into the `channel-provider` package.

Reply:
I think this makes sense but it's annoying/non-trivial enough to do that I'm going to punt and create a separate issue for it (#961).